.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,45 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:14
```

Comment:
We're using version 17 in other places. Should we use 17 here?

Reply:
yeah, i will go ahead and update that